github/ci/prow/files/jobs/hostpath-provisioner/OWNERS

```diff
@@ -0,0 +1,6 @@
+filters:
+  ".*":
+    reviewers:
+      - awels
```

Comment:
Best to add someone else  from your team too. Otherweise you will either have to configure prow for your repo, that authors of their PR can also /lgtm their own code, or you will have to rely on external reviewers too.

Reply:
Sure, I can add some people.